features/steps/steps.go

```diff
@@ -279,3 +287,24 @@ func (c *IdentityComponent) theResponseHeaderShouldContain(key, value string) (e
 
 	return nil
 }
+
+func (c *IdentityComponent) iShouldReceiveTheFollowingSortedJSONResponseWithStatus(body *godog.DocString) (err error) {
+	var expected, actual []models.UserParams
+	if err = json.Unmarshal([]byte(body.Content), &expected); err != nil {
+		return
+	}
+
+	responseBody := c.apiFeature.HTTPResponse.Body
+	resBody, err := io.ReadAll(responseBody)
+	if err != nil {
+		return err
+	}
+	fmt.Println("\n\t--- actual ----", resBody)
+	fmt.Println("\n\t--- expected ----", expected)
+
+	if err = json.Unmarshal(resBody, &actual); err != nil {
+		return err
+	}
+
+	return nil
+}
```

Comment:
do we actually need a separate function here? Could we just use the same step description and point to the normal 'receive the following json' response?

Reply:
Looking at the feature file, I don't think you're actually using this.